apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts

```diff
@@ -743,14 +733,21 @@ describe('getIndictmentInfo', () => {
 
     const indictmentInfo = getIndictmentInfo(
       CaseIndictmentRulingDecision.FINE,
-      rulingDate,
+      rulingDate.toISOString(),
       defendants,
     )
 
+    const expectedIndictmentAppealDeadline = new Date(
```

Comment:
I noticed that you are doing this in other files as well. Consider extracting this into a helper function. Something like 

`const endOfDay: Date = (day: Date) => ...`

Reply:
Yess, like it

Reply:
I ended up just using `endOfDay` from `date-fns`